packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -425,15 +463,23 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
               trailing: widget.trailing ?? _buildTrailingIcon(context),
             ),
           ),
-          ClipRect(
+       if (widget.child != null) Flexible(child: ClipRect(
             child: Align(
               alignment: widget.expandedAlignment
                 ?? expansionTileTheme.expandedAlignment
                 ?? Alignment.center,
               heightFactor: _heightFactor.value,
               child: child,
             ),
-          ),
+           ),) else ClipRect(
+            child: Align(
+              alignment: widget.expandedAlignment
+                ?? expansionTileTheme.expandedAlignment
+                ?? Alignment.center,
+              heightFactor: _heightFactor.value,
+              child: child,
+            ),
+           ),
```

Comment:
To avoid duplication, consider creating a local variable with the Align widget and its content.
Then simplify the if statement using this variable.

Reply:
refactored duplication